handlers.go

```diff
@@ -91,23 +91,24 @@ import (
 // @license.url https://github.com/application-research/estuary/blob/master/LICENSE.md
 
 // @host api.estuary.tech
-// @BasePath  /
+// @BasePath  /v1/
 // @securityDefinitions.Bearer
 // @securityDefinitions.Bearer.type apiKey
 // @securityDefinitions.Bearer.in header
 // @securityDefinitions.Bearer.name Authorization
 func (s *Server) ServeAPI() error {
-	e := echo.New()
-	e.Binder = new(util.Binder)
+	myecho := echo.New()
```

Comment:
can we rename this to `echoEngine` instead or any other name.

Reply:
pushed a commit with the rename, thanks for catching that!